katello/katello/hostname-change.rb

```diff
@@ -161,6 +161,16 @@ def get_default_proxy_id
       proxy["Id"]
     end
 
+    def custom_cert_options_check(scenario_answers)
+      if scenario_answers["certs"]["server_cert"] &&
+         scenario_answers["certs"]["server_key"] &&
+         scenario_answers["certs"]["server_cert_req"]
+        @using_custom_certs = true
```

Comment:
I don't like this side effect here. Maybe it should be ```custom_cert_options_check!```. It could also raise an exception if it's invalid.